mcli/mcli-1b-eval.yaml

```diff
@@ -1,7 +1,7 @@
 integrations:
 - integration_type: git_repo
   git_repo: mosaicml/llm-foundry
-  git_branch: v0.2.0
+  # git_branch: # Specify your git branch
```

Comment:
This still needs to be pinned, so if this yaml is updated to only work with main, we need to repin after this PR goes in.

Reply:
what should I pin it to?